.gitmessage.txt

```diff
@@ -0,0 +1,11 @@
+Subject line (try to keep under 50 characters)
+
+{current situation} -- use present tense}
+
+{why it needs to change}
+
+{what is being done about it} -- use imperative mood}
+
+{why it is done that way}
+
+{any other relevant info}
```

Comment:
perhaps can remove this file? this seems like a suggestion from CS2103T side but i don't really see it in my fork of iVolunteer 🤔

Reply:
I realised I need it cos they want us to write proper commit messages and I can't run `git commit` if I don't create this file first, not sure if yall face the same issue or not. Anyway I added it to gitignore